esp32-wroom-rp/src/tcp_client.rs

```diff
@@ -133,13 +176,12 @@ where
         self.protocol_handler.get_socket()
     }
 
-    // TODO: Make this non-public
-    /// Returns `Socket` reference set by calling `get_socket()`
+    /// Returns [`Socket`] reference set by calling [`TcpClient::get_socket`]
```

Comment:
We made this function private in #53 and it is not used internally. Maybe we should just remove it for now. Clippy seems to agree.